MM/5.10.0/config/media_manager_5/fields/page_footer.dcl

```diff
@@ -3,8 +3,8 @@ resource configservice_bit_config_field show_page_footer {
     product_id = resource.configservice_product.media_manager_5.id
     group = 'default'
     key = 'pageFooterEnabled'
-    title = 'Page footer'
-    description = 'If enabled, footer will appear at the bottom of the page'
+    title = 'Show page footer'
+    description = 'If enabled, a footer will be shown at the bottom of the asset list page'
```

Comment:
```suggestion
    description = 'If enabled, a footer will be shown at the bottom of the asset overview'
```

Reply:
I think "asset list" is not the correct term.
I tend to use "Asset overview".
"List" insinuates list view IMO. I.e., the grid view cannot be said to be a list.

You may choose to disregard this suggestion 👍